sections/semantics-forms.include

```diff
@@ -5390,8 +5416,9 @@ input[type="range"] { height: 75px; width: 49px; background: #D5CCBB; color: #00
     <{input/alt}>,
     <{input/autocapitalize}>,
     <{input/autocomplete}>,
+    <{input/capture}>,
     <{input/checked}>,
-    <{input/dirname}>,
+       <{input/dirname}>,
```

Comment:
Odd whitespace insertion - probably not critical (although sometimes whitespace oddities break the bikeshed builder).